assets/src/components/detours/diversionPanel.tsx

```diff
@@ -38,10 +38,7 @@ export const DiversionPanel = ({
           <h2 className="c-diversion-panel__h2">Affected route</h2>
 
           <div className="d-flex">
-            <RoutePill
-              className="d-inline-block me-2 align-top"
-              routeName={routeName}
-            />
+            <RoutePill className="me-2 align-top" routeName={routeName} />
```

Comment:
Curiosity: Was `d-inline-block` not doing anything?

Reply:
correct ✅